bin/riak-bucket-exporter.js

```diff
@@ -22,9 +23,16 @@ var count = 0;
 var openWrites = 0;
 var db = require("riak-js").getClient({host: program.host, port: program.port});
 var fs = require('fs');
+var deleteKeys = false;
```

Comment:
I think your version definitely looks better, but my thinking behind the later reassignment is that I only wanted it to be possible for `deleteKeys` to equal `true` when doing an export (never on import)